phpstan.neon

```diff
@@ -37,3 +37,14 @@ parameters:
         # Low value and frequent L5 errors that are ignored by default.
         - message: "#^Parameter \\#1 \\$.* of function array_multisort is passed by reference, so it expects variables only\\.$#"
         - message: '#^Parameter \#\d+ .* of (static )?(function|method) .* expects string(\|null)?, int.* given\.$#'
+
+        # TODO(jchaffraix): Temporarily disabling those as they are dynamically created (see comment in Pool's constructor).
```

Comment:
I am seeing those errors locally (which makes sense as we don't define those). As a lot can go wrong with this PR already, I would prefer to defer fixing those in a follow-up PR.